src/services/billing/cost-management/budget/budget-detail/modules/budget-info/BudgetDetailInfo.vue

```diff
@@ -15,11 +15,12 @@
                 Target
             </span>
             <p v-if="!loading" class="summary-content">
-                <p-anchor :to="referenceRouter(
-                    budgetData.project_id,
-                    { resource_type: 'identity.Project' })"
+                <p-anchor v-if="budgetData.project_group_id || budgetData.project_id"
+                          :to="referenceRouter(
+                              (budgetData.project_id ? budgetData.project_id : budgetData.project_group_id),
```

Comment:
```suggestion
                              budgetData.project_id ?? budgetData.project_group_id,
```

Reply:
아!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 저 진짜 이 문법이 갑자기 기억안나가지고 ㅠ 감사해요!!

Reply:
아 근데 이거 하니까 마크업에서 오류나서 ㅠ

Reply:
아맞다 template  내에선 타입스크립트 지원이 안되죠
아래처럼 하면 될거에요!

```suggestion
                              budgetData.project_id || budgetData.project_group_id,
```